js/ui/grid_core/ui.grid_core.header_panel.js

```diff
@@ -1,248 +1,2 @@
-                } else {
-                    toolbarInstance.option(itemOptionPrefix + '.disabled', optionValue);
-                }
-            }
-        }
-    },
-
-    updateToolbarDimensions: function() {
-        // @ts-expect-error
-        this._toolbar?.updateDimensions();
-    },
-
-    getHeaderPanel: function() {
-        return this.element();
-    },
-
-    getHeight: function() {
-        return this.getElementHeight();
-    },
-
-    optionChanged: function(args) {
-
-        if(args.name === 'onToolbarPreparing') {
-            this._invalidate();
-            args.handled = true;
-        }
-        if(args.name === 'toolbar') {
-            args.handled = true;
-            if(this._toolbar) {
-                const parts = getPathParts(args.fullName);
-                const optionName = args.fullName.replace(/^toolbar\./, '');
-
-                if(parts.length === 1) {
-                    // `toolbar` case
-                    const toolbarOptions = this._getToolbarOptions();
-                    this._toolbar.option(toolbarOptions);
-                } else if(parts[1] === 'items') {
-                    if(parts.length === 2) {
-                        // `toolbar.items` case
-                        const toolbarOptions = this._getToolbarOptions();
-                        this._toolbar.option('items', toolbarOptions.items);
-                    } else if(parts.length === 3) {
-                        // `toolbar.items[i]` case
-                        const normalizedItem = this._normalizeToolbarItems(this._getToolbarItems(), args.value);
-                        this._toolbar.option(optionName, normalizedItem);
-                    } else if(parts.length >= 4) {
-                        // `toolbar.items[i].prop` case
-                        this._toolbar.option(optionName, args.value);
-                    }
-                } else {
-                    // `toolbar.visible`, `toolbar.disabled` case
-                    this._toolbar.option(optionName, args.value);
-                }
-            }
-        }
-        this.callBase(args);
-    },
-
-    isVisible: function() {
-        return !!(this._toolbarOptions && this._toolbarOptions.visible);
-    },
-
-    allowDragging: noop,
-
-    hasGroupedColumns: noop,
-};
-
-const HeaderPanel = ColumnsView.inherit(members);
-
-/**
- * @type {import('./ui.grid_core.modules').Module}
- */
-export const headerPanelModule = {
-    defaultOptions: function() {
-        return {
-        };
-    },
-    views: {
-        headerPanel: HeaderPanel
-    },
-    extenders: {
-        controllers: {
-            resizing: {
-                _updateDimensionsCore: function() {
-                    this.callBase.apply(this, arguments);
-
-                    this.getView('headerPanel').updateToolbarDimensions();
-                }
-            }
-        }
-    }
-};
+/** @deprecated */
```

Comment:
Let's change the js-doc comment to the usual comment
```javascript
/** @deprecated */
```
to
```javascript
// deprected
```

Because js-doc comments can cause some strange behaviour in our metadata-tools :(